Crash in upload activity due to recent migrations to androidx

**Summary:** 

After the recent migrations to androidx upload activity crashes due to the following log message

**Steps to reproduce:** 

- Click on the FAB 
- Select a picture
- Upload

**System logs:**

```
Unable to start activity ComponentInfo{fr.free.nrw.commons.beta/fr.free.nrw.commons.upload.UploadActivity}: android.view.InflateException: Binary XML file line #23: Binary XML file line #70: Error inflating class androidx.support.v7.widget.CardView
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2791)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2856)
        at android.app.ActivityThread.-wrap12(ActivityThread.java)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1574)
        at android.os.Handler.dispatchMessage(Handler.java:110)
        at android.os.Looper.loop(Looper.java:203)
        at android.app.ActivityThread.main(ActivityThread.java:6364)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1076)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:937)
     Caused by: android.view.InflateException: Binary XML file line #23: Binary XML file line #70: Error inflating class androidx.support.v7.widget.CardView
     Caused by: android.view.InflateException: Binary XML file line #70: Error inflating class androidx.support.v7.widget.CardView
     Caused by: java.lang.ClassNotFoundException: Didn't find class "androidx.support.v7.widget.CardView" on path: DexPathList[[zip file "/data/app/fr.free.nrw.commons.beta-2/base.apk", zip file "/data/app/fr.free.nrw.commons.beta-2/split_lib_dependencies_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-2/split_lib_slice_0_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-2/split_lib_slice_1_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-2/split_lib_slice_2_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-2/split_lib_slice_3_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-2/split_lib_slice_4_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-2/split_lib_slice_5_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-2/split_lib_slice_6_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-2/split_lib_slice_7_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-2/split_lib_slice_8_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-2/split_lib_slice_9_apk.apk"],nativeLibraryDirectories=[/data/app/fr.free.nrw.commons.beta-2/lib/arm64, /data/app/fr.free.nrw.commons.beta-2/base.apk!/lib/arm64-v8a, /data/app/fr.free.nrw.commons.beta-2/split_lib_dependencies_apk.apk!/lib/arm64-v8a, /data/app/fr.free.nrw.commons.beta-2/split_lib_slice_0_apk.apk!/lib/arm64-v8a, /data/app/fr.free.nrw.commons.beta-2/split_lib_slice_1_apk.apk!/lib/arm64-v8a, /data/app/fr.free.nrw.commons.beta-2/split_lib_slice_2_apk.apk!/lib/arm64-v8a, /data/app/fr.free.nrw.commons.beta-2/split_lib_slice_3_apk.apk!/lib/arm64-v8a, /data/app/fr.free.nrw.commons.beta-2/split_lib_slice_4_apk.apk!/lib/arm64-v8a, /data/app/fr.free.nrw.commons.beta-2/split_lib_slice_5_apk.apk!/lib/arm64-v8a, /data/app/fr.free.nrw.commons.beta-2/split_lib_slice_6_apk.apk!/lib/arm64-v8a, /data/app/fr.free.nrw.commons.beta-2/split_lib_slice_7_apk.apk!/lib/arm64-v8a, /data/app/fr.free.nrw.commons.beta-2/split_lib_slice_8_apk.apk!/lib/arm64-v8a, /data/app/fr.free.nrw.commons.beta-2/split_lib_slice_9_apk.apk!/lib/arm64-v8a, /system/lib64, /vendor/lib64, /system/vendor/lib64]]
        at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:56)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:380)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
        at android.view.LayoutInflater.createView(LayoutInflater.java:615)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:832)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:772)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:903)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:866)
        at android.view.LayoutInflater.parseInclude(LayoutInflater.java:1039)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:899)
E/ACRA:     at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:866)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:906)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:866)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:524)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:432)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:383)
        at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:469)
        at androidx.appcompat.app.AppCompatActivity.setContentView(AppCompatActivity.java:141)
        at fr.free.nrw.commons.upload.UploadActivity.onCreate(UploadActivity.java:146)
        at android.app.Activity.performCreate(Activity.java:6695)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1124)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2744)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2856)
        at android.app.ActivityThread.-wrap12(ActivityThread.java)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1574)
        at android.os.Handler.dispatchMessage(Handler.java:110)
        at android.os.Looper.loop(Looper.java:203)
        at android.app.ActivityThread.main(ActivityThread.java:6364)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1076)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:937)
```

 
**Commons app version:** 
latest master


**Would you like to work on the issue?**

Yes

--- a/app/src/main/res/layout/activity_upload_bottom_card.xml
+++ b/app/src/main/res/layout/activity_upload_bottom_card.xml
@@ -67,7 +67,7 @@
     </androidx.cardview.widget.CardView>
 
 
-    <androidx.support.v7.widget.CardView
+    <androidx.cardview.widget.CardView
         android:id="@+id/bottom_card"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
@@ -177,8 +177,7 @@
                 app:layout_constraintStart_toStartOf="parent" />
 
         </androidx.constraintlayout.widget.ConstraintLayout>
-    </androidx.support.v7.widget.CardView>
+    </androidx.cardview.widget.CardView>
 </androidx.constraintlayout.widget.ConstraintLayout>
 
 
-                

